### PR TITLE
Feat/challenge funcions 1

### PR DESCRIPTION
**Pull request by HuevoPlatino**

### CODE REVIEW

**Comment by tictools** [comment]
👀 REVIEWED

File: exercise_01/getDistanceFromThreshold.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ * output: number
+ */
+/**
+ * 
+ * @param {number} value 
+ * @param {number} threshold 
+ * @returns number
+ */
+const getDistanceFromThreshold= function (value, threshold) {
+    const result = value - threshold;
```
Comment (Lye-Frozen-Heart) [inline exercise_01/getDistanceFromThreshold.js]: This could be just: -> returns value - threshold

File: exercise_01/getDistanceFromThreshold.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/**
+ * #1 :: Export JS function "getDistanceFromThreshold" to find how much a number exceeds or is left to reach one value
+ * input: number (value)
+ * input: number (threshold)
+ * output: number
+ */
+/**
+ *
```
Comment (Lye-Frozen-Heart) [inline exercise_01/getDistanceFromThreshold.js]: You need to describe the signature

File: exercise_02/getTotalWordsFromString.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+/**
+ * #2 :: Export JS function "getTotalWordsFromString" to get the total number of words contained in one string.
+ * input: string (sentence)
+ * output: number
+ */
+/**
+ *
```
Comment (Lye-Frozen-Heart) [inline exercise_02/getTotalWordsFromString.js]: You need to describe the function

File: exercise_02/getTotalWordsFromString.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+ * @param {number} sentence
+ * @returns number
+ */
+const getTotalWordsFromString = function (sentence) {
+  /**  @returns sentence.split(" ").length; 
+    const result = sentence.split(" ").length;
+    return result;
```
Comment (Lye-Frozen-Heart) [inline exercise_02/getTotalWordsFromString.js:14]: IMHO the comented code is a more eficient option

File: exercise_03/generateRandomNumberInRange.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+/**
+ * #3 :: Export JS function "generateRandomNumberInRange" to generate a rounded random number depending on a limit
+ * input: number (limit)
+ * output: number
+ */
+/**
+ *
```
Comment (Lye-Frozen-Heart) [inline exercise_03/generateRandomNumberInRange.js]: You need to describe the functions

File: exercise_04/isGreaterThan.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
+/**
+ * #4 :: Export JS function "isGreaterThan" to check whether one number is greater than a given number
+ * input: number (value)
+ * input: number (threshold)
+ * output: boolean
+ */
+/**
+ *
```
Comment (Lye-Frozen-Heart) [inline exercise_04/isGreaterThan.js]: You need to describe the function

File: exercise_04/isGreaterThan.js [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+ * output: boolean
+ */
+/**
+ * 
+ * @param {number} value 
+ * @param {number} threshold 
+ * @returns boolean
+ */
+const isGreaterThan = function (value, threshold) {
+    const result = value > threshold;
```
Comment (Lye-Frozen-Heart) [inline exercise_04/isGreaterThan.js]: This could be a one liner

File: exercise_05/isAdditionGreaterThanFifty.js [hunk 8]
```diff
@@ -0,0 +1,18 @@
+/**
+ * #5 :: Export JS function "isAdditionGreaterThanFifty" to check a pair of numbers and return true if their sum is 50 or greater than 50
+ * input: number (summand1)
+ * input: number (summand2)
+ * output: boolean
+ */
+/**
+ *
```
Comment (Lye-Frozen-Heart) [inline exercise_05/isAdditionGreaterThanFifty.js]: Describe the function

File: exercise_05/isAdditionGreaterThanFifty.js [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+ * output: boolean
+ */
+/**
+ * 
+ * @param {number} summand1 
+ * @param {number} summand2 
+ * @returns boolean
+ */
+const isAdditionGreaterThanFifty = function (summand1, summand2) {
+    const result = summand1 + summand2 > 50;
```
Comment (Lye-Frozen-Heart) [inline exercise_05/isAdditionGreaterThanFifty.js]: A one liner would be more eficient

File: exercise_06/checkCollectionHasElements.js [hunk 10]
```diff
@@ -0,0 +1,16 @@
+/**
+ * #6 :: Export JS function "checkCollectionHasElements" to get length of a collection and check whether it is empty or not
+ * input: array (collection)
+ * output: boolean
+ */
+/**
+ *
```
Comment (Lye-Frozen-Heart) [inline exercise_06/checkCollectionHasElements.js]: Describe the funtion

File: exercise_07/isTypeOf.js [hunk 11]
```diff
@@ -6,3 +6,19 @@
  * input: boolean (type)
  * output:
  */
+/**
+ *
```
Comment (Lye-Frozen-Heart) [inline exercise_07/isTypeOf.js]: Describe the function.

File: exercise_07/isTypeOf.js [hunk 12]
```diff
@@ -6,3 +6,19 @@
...
  * output:
  */
+/**
+ * 
+ * @param {string | number | boolean | array } value 
+ * @param {string} type 
+ * @returns booelan
+ */
+const isTypeOf = function (value, type) {
+  if (type === "array") {
```
Comment (Lye-Frozen-Heart) [inline exercise_07/isTypeOf.js:16]: This if should consider the passing of Array.isArray() too.

**Review by Lye-Frozen-Heart — APPROVED** [review]

**Review by tictools — APPROVED** [review]

**Pull request merged**